doc/faq/a02-my-esp-crashes.rst

```diff
@@ -286,6 +286,9 @@ Memory, memory, memory
    modification. For example, std::vector is available for use. The standard implementations
    rely on exceptions for error handling, which is not available for the ESP, and in any
    case there is no access to the underlying code.
+   
+   Instrumenting the code with the OOM debug option and calls to ``ESP.getFreeHeap()`` will
```

Comment:
Should we add a line about the last memory allocation failure report in the panic that was added a while back?